.circleci/config.yml

```diff
@@ -71,6 +71,7 @@ jobs:
       - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
       - run: npm ci
       - run: npm run db:setup
+      - run: npm run db:migrate
```

Comment:
What is the impact of the migration? It was skipped on purpose, the model is supposedly already up to date, isn't it?

Reply:
Without migration sequelize.sync does not find the tables in postgres and displays the error before creating each one. That's why running the migrations avoids the error log in the tests.

Reply:
Thank you, I'll dig deeper into it.

Reply:
So, after investigation, the outcome is great, we do want to have the schema created before running the tests.

However, I don't think it's good idea and create the schema with `npm run db:migrate`, it's slow and ineffective. I think it's better to use `sequelize.sync` and add it in the setup script.

I explored this solution in: https://github.com/opencollective/opencollective-api/pull/2973